docs/aws-professional/services.md

```diff
@@ -5,6 +5,10 @@ description: See how Microsoft Azure cloud services compare to Amazon Web Servic
 keywords: cloud services comparison, cloud services compared, multicloud, compare azure aws, compare azure and aws, compare aws and azure, IT capabilities
 author: lbrader
 ms.date: 11/09/2018
+ms.topic: article
+ms.service: architecture-center
+ms.subservice: cloud-fundamentals
+ms.custom: networking, data-analytics, IoT, identity
```

Comment:
let's omit the ms.custom tag on this file.

Reply:
@david-stanford The ms.custom tag has been removed from docs/aws-professional/services.md .
Thanks - mw